frontend/src/components/notifications/NotificationBell.jsx

```
@@ -0,0 +1,43 @@
+  };
+  useEffect(() => {
+    fetchUnreadCount();
+  }, [userId]);
+
+  return (
+    <div className="relative inline-block">
+      <button
+        onClick={() => setDropdownOpen(!dropdownOpen)}
+        className="relative"
+      >
+        <span className="material-icons">Notifications</span>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)